### DISCUSSION TITLE
[Script request] FreshRSS

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#608] [Request script]
### Application Name

FreshRSS

### Website

https://github.com/FreshRSS

### Description

 FreshRSS is a self-hosted RSS and Atom feed aggregator.
It is lightweight, easy to work with, powerful, and customizable.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Duplicate of: https://github.com/community-scripts/ProxmoxVE/discussions/31

**Reply by Mightyretro** [reply]
Thank, forget to check
